docs/diagrams/commands/ViewProfileCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,31 @@
+@startuml
+!include ../style.puml
+
+box Logic LOGIC_COLOR_T1
+Participant ":LogicManager" as logicManager LOGIC_COLOR_T3
+Participant ":NuSchedulerParser" as nuSchedulerParser LOGIC_COLOR_T3
+Participant "p:ViewProfileCommand" as viewProfileCommand LOGIC_COLOR_T3
+Participant "r:CommandResult" as commandResult LOGIC_COLOR_T3
+endBox
+
+box Model MODEL_COLOR_T1
+Participant "model:Model" as model MODEL_COLOR_T3
+Participant "filteredProfiles:FilteredList" as filteredList MODEL_COLOR_T3
+endBox
+
+-> logicManager ++: execute("profile -v")
+logicManager -> nuSchedulerParser ++: parseCommand
+return p
+logicManager -> viewProfileCommand ++: execute(model)
+viewProfileCommand -> model ++: updateFilteredProfileList(predicate)
+model -> filteredList ++: setPredicate(predicate)
+return
```

Comment:
Maybe the `ViewProfileCommand` should be created somewhere here instead, to better reflect the lifeline of the `ViewProfileCommand`?

Reply:
Yea I was thinking of lowering the `ViewProfileCommand`, but I don't want to show what happens during the `NuSchedulerParser`, so means I won't calling the constructor of `ViewProfileCommand` here. But I can't seem to get the `ViewProfileCommand` box to be lowered to show that it doesn't exist as earlier as eg. the `NuSchedulerParser`.

Does anyone know how to do this?

Reply:
You can refer to my other comment on how to do this. 👍🏼 .